intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -0,0 +1,162 @@
+- I'm making an `Album` object. The `Album` object is the "Purple Rain" album.
+- The "Purple Rain" instance of `Album` is very different from the "Dirty Computer" instance of `Album`.
+- "Purple Rain" and "Dirty Computer" are both instances of `Album`.
+
+### State and Behavior
+
+When we start defining our own classes, we will define their generic **state** and their generic **behavior.**
+
+**State** describes any data or attributes that should be kept over its lifetime.
+
+**Behavior** describes any actions that objects of this class should be able to do.
+
+### !callout-info
+
+## Examples: State and Behavior
+
+Observe these examples of classes that already exist, or classes that could exist.
+
+| Class  | Instances                                                                                                                 | Something Kept in State                                                         | Behavior it Can Do                                                           |
+| ------ | ------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| String | `"Hello World! I am an instance of a String!"`                                                                            | These specific characters, in this specific order                               | Become CAPITALIZED if it needs to                                            |
+| List   | `["red", "orange", "yellow"]`                                                                                             | These specific items, in this specific order                                    | Reverse the order of its elements                                            |
+| Driver | A driver with ID `DR0004`, 3 trips, who has earned $35, has an average rating of 4.67                                     | Its ID, number of trips, total amount of money earned, and total average rating | Calculate its own average rating, tell you the number of trips they've taken |
+| Album  | Dirty Computer (2018), which has 14 tracks in a certain order, was released April 2018, has a total album length of 48:42 | Tracks (number and order), release date, total album length                     | Return the audio data for any specific track it has                          |
+
+### !end-callout
+
+## Summary
+
+In OOP...
+
+We will represent an idea as a data type by defining a **class.** A class is a template, or blueprint of the idea. The class is where we define the idea's generic **state** and **behavior.**
+
+After we've defined the idea by defining a **class**, we will use this template/blueprint to create manifestations, or **instances** of this class. These instances are also known as **objects.**
```

Comment:
Not important (still gonna go with the above suggestion as-is), but I wish there were maybe a different word besides "examples." Just speaking this into the air.

Reply:
I know, right!? But I could not find a single synonym that I was happy with.  It's super annoying that the most appropriate normal English word to use would just be "instance", but that's a little weird since we're defining instance.  A rare example where a programming term and the actual English word align in meaning...